From 21dc84d9a592cad908d5dd77a8d0d314be565434 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Fri, 12 Aug 2016 12:33:05 +0200
Subject: [PATCH 126/188] phy: zynqmp: Add missing header to declare exported
 functions

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

This patch removes sparse warnings:
drivers/phy/phy-zynqmp.c:311:5: warning: symbol 'xpsgtr_override_deemph'
was not declared. Should it be static?
drivers/phy/phy-zynqmp.c:328:5: warning: symbol
'xpsgtr_margining_factor' was not declared. Should it be static?
drivers/phy/phy-zynqmp.c:647:5: warning: symbol 'xpsgtr_wait_pll_lock'
was not declared. Should it be static?

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit 3f30976991ea7d48fe55b70051de406659945e08)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/phy/phy-zynqmp.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/phy/phy-zynqmp.c b/drivers/phy/phy-zynqmp.c
index 41be9f1..882553c 100644
--- a/drivers/phy/phy-zynqmp.c
+++ b/drivers/phy/phy-zynqmp.c
@@ -29,6 +29,7 @@
 #include <linux/of_platform.h>
 #include <linux/of_address.h>
 #include <linux/phy/phy.h>
+#include <linux/phy/phy-zynqmp.h>
 #include <linux/platform_device.h>
 #include <linux/delay.h>
 #include <dt-bindings/phy/phy.h>
-- 
2.0.2

